CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - `mediaType` option for `CadenzaClient#show`
 - `CadenzaClient#fetchData`
 - `CadenzaClient#downloadData`
+- `parts` option for `CadenzaClient#fetchData` and `CadenzaClient#downloadData`
```

Comment:
Since the whole #fetchData and #downloadData methods are new, imo it's not necessary to have an extra line in the changelog for this new option.

Reply:
done